style.css

```diff
@@ -0,0 +1,514 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.container {
+  background: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.part1 {
+  border: 1px solid rgb(202, 199, 199);
+  background: url(images/background.jpeg);
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  align-items: center;
+  margin-bottom: 2%;
+}
+
+.head-name {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 700;
+  margin: 40% 10px 17px;
+  line-height: 50px;
+  right: 10px;
+  display: flex;
+}
+
+.head-para {
+  font-family: 'Poppins', sans-serif;
+  left: 20px;
+  right: 10px;
+  margin: 10px 17px;
+}
+
+.sub-head {
+  font-family: 'Poppins', sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  margin-left: -48%;
+  color: #7f8cff;
+  display: flex;
+}
+
+.social-media ul {
+  color: black;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  font-size: 20px;
+  padding: 0 10px;
+  margin-left: -30%;
+}
+
+.social-media ul li {
+  text-decoration: none;
+  padding: 0 12px;
+}
+
+.logo {
+  position: absolute;
+  margin: 10px;
+  top: 1%;
+  font-size: 24px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  gap: 12px;
+  margin-top: 86px;
+  margin-right: 10px 10px 10px 10px;
+  width: 100%;
+  height: auto;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding-top: 20px;
+  left: 24px;
+  top: 10%;
+  padding-bottom: 30px;
+}
+
+.card img {
+  width: 100%;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.card .instruction {
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  top: 94px;
+}
+
+.card .btn {
+  height: 40px;
+  color: #396df2;
+  font-weight: 600px;
+  font-size: 17px;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 10px 14px;
+  border: 1.5px solid #396df2;
+  width: fit-content;
+  margin-top: -34px;
+  margin-left: 20px;
+}
```

Comment:
- [ ] kindly consider following the button styling: Active, Hover and Pressed as given in the design 👇 consider adding any of them 
![tt](https://user-images.githubusercontent.com/60146030/221375982-b65e681a-1cfd-4a85-9a81-4579a4b8a945.PNG)